hstest/django_test.py

```diff
@@ -50,6 +63,11 @@ def __find_free_port(self):
                 if isinstance(err.reason, ConnectionRefusedError):
                     self.port = port
                     break
+            except ConnectionResetError:
+                pass
+
+    def read_page(self, link: str) -> str:
+        return urlopen(link).read().decode().replace('\u00a0', ' ')
```

Comment:
What is `\u00a0` symbol?

Reply:
NO-BREAK SPACE or nbsp;
https://www.fileformat.info/info/unicode/char/00A0/index.htm

Reply:
There are a lot of invisible spaces, why to replace one and not to replace others then?
https://en.wikipedia.org/wiki/Whitespace_character

Reply:
I encounter problems with `nbsp;`, because it's common for web and in particularly it was in one of the tasks. When I copypaste a phrase from a task explanation I received an error message. It was hard to debug, what had gone wrong.

I don't know regex to match only different spaces, but we can replace `'\s'` to `' '` though.

Reply:
Ok, let's handle the problems as they occur and now pay attention only to `nbsp;`